lib/webdrivers/chromedriver.rb

```diff
@@ -31,18 +35,57 @@ def downloads
         Webdrivers.logger.debug "Versions previously located on downloads site: #{@downloads.keys}" if @downloads
 
         @downloads ||= begin
-          doc = Nokogiri::XML.parse(get(base_url))
+          doc   = Nokogiri::XML.parse(get(base_url))
           items = doc.css("Contents Key").collect(&:text)
-          items.select! {|item| item.include?(platform)}
+          items.select! { |item| item.include?(platform) }
           ds = items.each_with_object({}) do |item, hash|
-            key = normalize item[/^[^\/]+/]
+            key       = normalize item[/^[^\/]+/]
             hash[key] = "#{base_url}/#{item}"
           end
           Webdrivers.logger.debug "Versions now located on downloads site: #{ds.keys}"
           ds
         end
       end
 
+      # Returns release version from the currently installed Chrome version
+      #
+      # @example
+      #   73.0.3683.75 -> 73.0.3683
+      def release_version
+        chrome_version.scan(/\d+\.\d+\.\d+/)[0]
```

Comment:
We should avoid scan if we don't need the Array: `chrome_version[/\d+\.\d+\.\d+\.\d+/]`
Not sure if rubocop would flag this (https://github.com/rubocop-hq/ruby-style-guide#regexp-string-index)

Reply:
Oh nice! I didn't know we could do that. Updated to `chrome_version[/\d+\.\d+\.\d+/]` to get the first 3 versions.